tests/test_routes.py

```diff
@@ -1,15 +1,14 @@
 import unittest
-from webapp.wsgi import application
-from django.test import Client
+from webapp.app import app
 
 
 class TestRoutes(unittest.TestCase):
     def setUp(self):
         """
         Set up Flask app for testing
         """
-        self.app = application
-        self.client = Client()
+        app.testing = True
```

Comment:
Maybe `app.config["TESTING"]` is the more modern way to do this?

https://flask.palletsprojects.com/en/1.1.x/testing/#the-testing-skeleton

> During setup, the TESTING config flag is activated. What this does is disable error catching during request handling, so that you get better error reports when performing test requests against the application.

Reply:
done